tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -0,0 +1,164 @@
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+"""Test module for DELETE Object Tagging"""
+
+import logging
+import os
+import time
+
+import pytest
+
+from commons.ct_fail_on import CTFailOn
+from commons.errorcodes import error_handler
+from commons.params import TEST_DATA_FOLDER
+from commons.utils import assert_utils
+from commons.utils.system_utils import create_file, path_exists, remove_file
+from commons.utils.system_utils import make_dirs, remove_dirs
+from config.s3 import S3_CFG
+from libs.s3.s3_tagging_test_lib import S3TaggingTestLib
+from libs.s3.s3_test_lib import S3TestLib
+from libs.s3.s3_versioning_common_test_lib import put_object_tagging
+from libs.s3.s3_versioning_common_test_lib import upload_version
+from libs.s3.s3_versioning_test_lib import S3VersioningTestLib
+
+# Global Constants
+LOGGER = logging.getLogger(__name__)
+
+
+class TestTaggingDeleteObject:
+    """Test Delete Object Tagging"""
+
+    def setup_method(self):
+        """
+        Function will be invoked perform setup prior to each test case.
+        """
+        LOGGER.info("STARTED: Setup operations")
+        self.s3_test_obj = S3TestLib(endpoint_url=S3_CFG["s3_url"])
+        self.s3_ver_obj = S3VersioningTestLib(endpoint_url=S3_CFG["s3_url"])
+        self.s3_tag_obj = S3TaggingTestLib(endpoint_url=S3_CFG["s3_url"])
+        self.test_dir_path = os.path.join(TEST_DATA_FOLDER, "TestTaggingDeleteObject")
+        if not path_exists(self.test_dir_path):
+            make_dirs(self.test_dir_path)
+            LOGGER.info("Created path: %s", self.test_dir_path)
+        self.file_path = os.path.join(self.test_dir_path, f"del_obj_tag_{time.perf_counter_ns()}")
+        create_file(fpath=self.file_path, count=1)
+        LOGGER.info("Created file: %s", self.file_path)
+        self.bucket_name = f"tag-bkt-{time.perf_counter_ns()}"
+        self.object_name = f"tag-obj-{time.perf_counter_ns()}"
+        res = self.s3_test_obj.create_bucket(self.bucket_name)
+        assert_utils.assert_true(res[0], res[1])
+        assert_utils.assert_equal(res[1], self.bucket_name, res[1])
+        LOGGER.info("Created a bucket with name : %s", self.bucket_name)
+
+    def teardown_method(self):
+        """
+        Function will be perform cleanup after each test case.
+        """
+        LOGGER.info("STARTED: Teardown operations")
+        LOGGER.info("Clean : %s", self.test_dir_path)
+        if path_exists(self.file_path):
+            res = remove_file(self.file_path)
+            LOGGER.info("cleaned path: %s, res: %s", self.file_path, res)
+        if path_exists(self.test_dir_path):
+            remove_dirs(self.test_dir_path)
+        LOGGER.info("Cleanup test directory: %s", self.test_dir_path)
+        # DELETE Object with VersionId is WIP, uncomment once feature is available
```

Comment:
Reason of keeping this commented code?

Reply:
> Reason of keeping this commented code?

The feature is not supported yet. Please correct me @apoorva-rao if this differ